sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -142,8 +142,6 @@ private[sql] class HiveSessionCatalog(
           // let's try to load it as a Hive's built-in function.
           // Hive is case insensitive.
           val functionName = funcName.unquotedString.toLowerCase(Locale.ROOT)
-          logWarning("Encountered a failure during looking up function:" +
```

Comment:
Why don't we debug or info that we'll try fallback while we're here?

Reply:
Yes. Of course, we can add debug messages any place. But, this is a natural flow for Hive build-in function lookup. To showing a message like `Encountered a failure during looking up function: NoSuchFunctionException` for every Hive function at the first invocation will not give much values to the developer/users. Also, it's misleading to the users because Spark will execute the function without any problem at the same exeuction.

Reply:
I meant to add a debug message like "Hive fallback will be attempted for the function [abc]." because users/dev might not know which function exist in Spark and which function comes from Hive or at least where the fallback happens.

This warn should be removed anyway ..

Reply:
let's talk about this somewhere else later. I think it's not a big deal at all. LGTM it should be removed anyway.

Reply:
I see, @HyukjinKwon .
For that message, I guess that the reason why we don't have `debug/info` originally is that the Apache Spark's ambition was to implement everything in Spark-side and to remove this fallback logic completely. :)